platforms/stellar-disbursement-platform/admin-guide/30-getting-started.mdx

```diff
@@ -163,7 +154,14 @@ Click the Sign in button and the SDP Dashboard will open. You will have no disbu
 
 ## Create Your First Disbursement
 
-Create your first disbursement by clicking the New Disbursement button. Use Demo Wallet as your wallet and USDC as your asset. You can choose whatever values you like for Country and Name, but ensure that Verification type matches the type used in the disbursement file that you will then upload with receiver information. Refer to `./dev/sample/sample-disbursement.csv` for the disbursement file template. ![Disbursement Details](/assets/SDP/SDP28.png)
+Navigate to the frontend service by opening a browser and going to http://bluecorp.stellar.local:3000.
+
+- Click `New Disbursement+` on the Dashboard screen.
+- Use `Demo Wallet` as your wallet and choose a verification method.
+- Upload a disbursement file. The file format can vary according with the disbursement configuration and a few sample files are available at [public/resources/disbursementTemplates](https://github.com/stellar/stellar-disbursement-platform-frontend/tree/58873bbf36cff4614e603daf449079b1d9fad24a/public/resources/disbursementTemplates). Make sure to update the file before using it by changing the contact info to yours and removing the comment line.
```

Comment:
Maybe it's worth mentioning that the correct template that matches a certain config can be directly downloaded from the frontend after we make those selections.

Reply:
https://github.com/stellar/stellar-docs/pull/1093/commits/73e2ac58217f3fdf8de0f2cac37358817e22867f